chat.py

```diff
@@ -17,6 +18,23 @@
 COST_ALERT_EMOJI = CONFIG['chat']['cost_alert_emoji'].get()
 COST_MIN_TO_NOTIFY = CONFIG['chat']['cost_min_to_notify'].get(float)
 
+NUMBER_OF_EXPENSIVE_PROJECTS = 0
+NUMBER_OF_NOTIFIED_PROJECTS = 0
```

Comment:
Make sure these variables are initialized every time the `send_message` method is called. Otherwise they may not be correctly updated.

I think assigning `0` again is sufficient to fix this.

Reply:
Change applied by commit [03fc0f9](https://github.com/kieras/zombie-projects-watcher/pull/63/commits/03fc0f9a35989755f89e998a633fb9c82cfb5978)